src/sentry/migrations/0546_migrate_no_action_dupe_issue_alerts.py

```diff
@@ -0,0 +1,81 @@
+# Generated by Django 3.2.20 on 2023-09-08 20:31
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def has_duplicate_rule(apps, rule_data, project, rule_id):
+    Rule = apps.get_model("sentry", "Rule")
```

Comment:
Is this the right way to do it again? I need to compare the rule to other ones in the same project

Reply:
also this function is basically https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_rules.py#L38 but modified slightly for the migration to return a boolean since we don't need the duplicated rule

Reply:
Yes, this is the correct way to get a reference to model classes in migrations.